### PR TITLE
Removed dynamic validations

### PR DESCRIPTION
**Pull request by khenrichsen**
Removing the dynamic validation for totalAmount, totalTax and totalDiscount

### CODE REVIEW

File: docs/swagger.yaml [hunk 1]
```diff
@@ -470,31 +467,31 @@ components:
...
-          description: Total tax for the order
+          description: 'Deprecated, tax will be calculated based on the sum of the orderLines'
           nullable: true
           minimum: 0
           format: int64
+          deprecated: true
         totalDiscount:
           type: integer
-          description: Total discount from the orderLines
+          description: 'Deprecated, discount will be calculated based on the sum of the orderLines'
```
Comment (annemsa75) [inline docs/swagger.yaml:490]: Deprecated, discount will be calculated based on the sum of the discount field on each orderLine?

File: docs/swagger.yaml [hunk 2]
```diff
@@ -470,31 +467,31 @@ components:
...
           type: integer
-          description: 'The total amount of the order, including all taxes, discounts and shipping. Can be negative'
+          description: 'Deprecated, sum will be calculated based on the sum of the orderLines'
           format: int64
           nullable: true
+          deprecated: true
         totalTax:
           type: integer
-          description: Total tax for the order
+          description: 'Deprecated, tax will be calculated based on the sum of the orderLines'
```
Comment (annemsa75) [inline docs/swagger.yaml:483]: Deprecated, tax will be calculated based on the sum of the totalTaxAmount field on each orderLine

File: docs/swagger.yaml (hunk 1 above)
Comment (annemsa75) [reply docs/swagger.yaml:490]: Bedre?

**Review by kaffikaffi — APPROVED** [review]
looks good

File: docs/swagger.yaml (hunk 1 above)
Comment (khenrichsen) [reply docs/swagger.yaml]: Jepp! Oppdatert

**Pull request merged**